actionview/test/template/form_helper/form_with_test.rb

```diff
@@ -365,6 +365,18 @@ def test_form_with
     assert_dom_equal expected, output_buffer
   end
 
+  def test_form_with_button_yields_translation
+    form_with(model: @post) do |f|
+      concat f.button { |value| concat(value) }
```

Comment:
Should use add a new tag inside the block to make it clear that this is not only the default button translation?